AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 balcaorural.com.br##.j-cookieAlert
+economist.com###footer-cookie-policy
```

Comment:
It hides only a link to cookie policy in the footer on my side.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/110124280-2d2d4100-7dc2-11eb-911a-6158cdd703b1.png)

</details>

I will add a different rule.